src/site/content/en/blog/pwas-on-oculus-2/index.md

```diff
@@ -0,0 +1,348 @@
+---
+layout: post
+title: 'PWAs on Oculus Quest&nbsp;2'
+authors:
+  - thomassteiner
+hero: image/8WbTDNrhLsU0El80frMBGE4eMCD3/B7zNhOVvzK3O71sQyLKe.jpg
+alt:
+  'Person wearing an Oculus Quest 2 device with a PWA sticker on it spreading their arms with Quest
+  controllers in both hands.'
+subhead: >
+  The Oculus Quest 2 is a virtual reality (VR) headset created by Oculus, a division of Meta.
+  Developers can now build and distribute 2D Progressive Web Apps that take advantage of Oculus
+  Quest&nbsp;2's multitasking feature.
+description: >
+  The Oculus Quest 2 is a virtual reality (VR) headset created by Oculus, a division of Meta.
+  Developers can now build and distribute 2D Progressive Web Apps that take advantage of Oculus
+  Quest 2's multitasking feature. This article describes the experience and how to build, sideload,
+  and test your PWA on the Oculus Quest 2.
+date: 2021-12-21
+tags:
+  - blog
+  - capabilities
+  - progressive-web-apps
+---
+
+## The Oculus Quest&nbsp;2
+
+The [Oculus Quest&nbsp;2](https://www.oculus.com/quest-2/) is a virtual reality (VR) headset created
+by Oculus, a division of Meta. It is the successor to the company's previous headset, the Oculus
+Quest. The device is capable of running as both a standalone headset with an internal, Android-based
+operating system, and with Oculus-compatible VR software running on a desktop computer when
+connected over USB or Wi-Fi. It uses the Qualcomm
+[Snapdragon XR2](https://www.qualcomm.com/news/onq/2020/10/29/oculus-quest-2-how-snapdragon-xr2-powers-next-generation-vr)
+system on a chip with 6&nbsp;GB of RAM. The Quest&nbsp;2's display is a singular fast-switch LCD
+panel with a 1,832&nbsp;×&nbsp;1,920 pixels per eye resolution, which can run at a refresh rate of
+up to 120&nbsp;Hz.
+
+{% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/g0IQmlLaOiLWqwQuvnhQ.jpeg", alt="Oculus Quest 2 device with controllers.", width="800", height="304" %}
+
+## The Oculus Browser
+
+Currently there are two browsers available for the Oculus Quest&nbsp;2,
+[Firefox Reality](https://www.oculus.com/experiences/quest/2180252408763702/) and the built-in
+[Oculus Browser](https://www.oculus.com/experiences/quest/1916519981771802). This article focuses on
+the latter. The Oculus website
+[introduces](https://developer.oculus.com/documentation/web/browser-intro/) the Oculus Browser as
+follows.
+
+_"Oculus Browser provides support for the latest web standards and other technologies to help you
+create VR experiences on the web. Today's 2D web sites work great in Oculus Browser because it's
+powered by the Chromium rendering engine. It's further optimized for Oculus headsets to get the best
+performance and to enable web developers take advantage of the full potential of VR with new APIs,
+like WebXR. Through WebXR, we're opening the doors to the next frontier of the web."_
```

Comment:
Should this be a blockquote? Seems like it ought to be.

Reply:
It should be, but unfortunately it looks horrible. It's not meant for long quotes like that, but short inspirational quotes like "Be yourself; everyone else is already taken". Maybe a feature request?

Reply:
I agree. The blockquote macro is designed more as a callout than as something to run in the main flow of text.